addons/mass_mailing_sms/views/mailing_mailing_views.xml

```diff
@@ -60,113 +58,258 @@
-                <attribute name="attrs">{'invisible': ['|',('mailing_type', '!=', 'mail'),('state', 'in', ('draft','test'))]}</attribute>
-            </xpath>
-            <xpath expr="//button[@name='action_view_replied']" position="attributes">
-                <attribute name="attrs">{'invisible': ['|',('mailing_type', '!=', 'mail'),('state', 'in', ('draft','test'))]}</attribute>
-            </xpath>
-            <!-- Form -->
-            <xpath expr="//field[@name='subject']" position="attributes">
-                <attribute name="attrs">{'invisible': [('mailing_type', '!=', 'mail')], 'readonly': [('state', 'in', ('sending', 'done'))], 'required': [('mailing_type', '=', 'mail')]}</attribute>
-                <!-- overrided in xml view to prevent remaining helper changes (on mass_mailing module) when mass_mailing_sms uninstalled-->
-                <attribute name="help">For an Email, Subject your Recipients will see in their inbox.
-                    For an SMS Text Message, internal Title of the Message.</attribute>
-            </xpath>
-            <xpath expr="//field[@name='subject']" position="after">
-                <field class="o_text_overflow" name="sms_subject" string="Title" placeholder="e.g. Black Friday SMS coupon" attrs="{'invisible': [('mailing_type', '!=', 'sms')], 'readonly': [('state', 'in', ('sending', 'done'))], 'required': [('mailing_type', '=', 'sms')]}"/>
-            </xpath>
-            <xpath expr="//field[@name='preview']" position="attributes">
-                <attribute name="attrs">{'readonly': [('state', 'in', ('sending', 'done'))], 'invisible': [('mailing_type', '!=', 'mail')]}</attribute>
-            </xpath>
-            <xpath expr="//page[@name='mail_body']" position="attributes">
-                <attribute name="attrs">{'invisible': [('mailing_type', '!=', 'mail')]}</attribute>
-            </xpath>
-            <xpath expr="//page[@name='mail_body']" position="after">
-                <page string="SMS Content" name="sms_body" attrs="{'invisible': [('mailing_type', '!=', 'sms')]}">
-                    <field name="body_plaintext" widget="sms_widget"
-                        attrs="{'required': [('mailing_type', '=', 'sms')], 'readonly': [('state', 'in', ('sending', 'done'))]}"
-                        options='{"enable_emojis": True}'/>
-                </page>
-            </xpath>
-            <xpath expr="//field[@name='user_id']" position="after">
-                <field name="sms_allow_unsubscribe" attrs="{'invisible': [('mailing_type', '!=', 'sms')], 'readonly': [('state', 'in', ('sending', 'done'))]}"/>
-            </xpath>
+                <div class="alert alert-info text-center" role="alert" attrs="{'invisible': ['&amp;','&amp;','&amp;','&amp;',('state', '!=', 'in_queue'),('sent', '=', 0),('ignored', '=', 0),('scheduled', '=', 0),('failed', '=', 0)]}">
+                    <div attrs="{'invisible': [('ignored', '=', 0)]}">
```

Comment:
Why don't you set the invisible directly on button ? It seems the div only adds unnecessary layer. In which case, check to remove other occurences, purpose being to have a layout as flat and as standard (without "div") as possible.